public/styles.css

```diff
@@ -199,22 +195,7 @@ section#pageThree {
     display: flex;
     justify-content: center;
 } */
-#navbar {
-    z-index: 11;
-    position: fixed;
-    background-color: #fff;
-    width: 100%;
-    top: 0;
-    border-bottom: 1px solid #324da6;
-    text-align: center;
-}
-#side-navbar {
-    z-index: 11;
-    position: fixed;
-    background-color: #fff;
```

Comment:
I think the White nav-bar had a better look. However if it was impeding responsiveness then I'm fine with it.

Reply:
Adding to that, quick suggestion, you can always go with `background-color: transparent` if needed.

Reply:
I am not sure I understand this comment @shalarewicz

Reply:
The toolbar on the lefthand side used to have a white background whereas now it's just the arrow icon on a dark background. I think the contrast for the toolbar looked better.

Reply:
Ohhh now I understand that. Gotcha. Althou it will have to stay like this for mobile phones due to visualization. The change will be made in mp/main-responsive

Reply:
To be addressed in #86